src/text/text_editor.py

```diff
@@ -194,54 +234,118 @@ def __pt_container__(self):
 
 
 @bindings.add("c-k")
-def _(event: object) -> None:
-    """Focus menu."""
+def open_menu(event: KeyPressEvent) -> None:
+    """Focus menu with Ctrl-K"""
     event.app.layout.focus(root_container.window)
 
 
 @bindings.add("escape")
-def _(event: object) -> None:
+def close_menu(event: KeyPressEvent) -> None:
     """Focus text field."""
     event.app.layout.focus(text_field)
 
 
+@bindings.add("c-n")
+def create_new_file(event: KeyPressEvent) -> None:
+    """Create new file with Ctrl-N"""
+    do_new_file()
+
+
+@bindings.add("c-s")
+def save_file(event: KeyPressEvent) -> None:
+    """Save file with Ctrl-S"""
+    do_save_file()
+
+
+@bindings.add("c-o")
+def open_file(event: KeyPressEvent) -> None:
+    """Open file with Ctrl-O"""
+    do_scroll_menu()
+
+
+@bindings.add("c-q")
+def exit_editor(event: KeyPressEvent) -> None:
+    """Exit terminal with Ctrl-Q"""
+    do_exit()
+
+
+@bindings.add("c-a")
+def select_all(event: KeyPressEvent) -> None:
+    """Select all with Ctrl-A"""
+    do_select_all()
+
+
+@bindings.add("c-x")
+def cut_text(event: KeyPressEvent) -> None:
+    """Cut with Ctrl-X"""
+    do_cut()
+
+
+@bindings.add("c-c")
+def copy_text(event: KeyPressEvent) -> None:
+    """Copy with Ctrl-C"""
+    do_copy()
+
+
+@bindings.add("c-v")
+def paste_text(event: KeyPressEvent) -> None:
+    """Paste with Ctrl-V"""
+    do_paste()
+
+
+@bindings.add("c-z")
+def undo_changes(event: KeyPressEvent) -> None:
+    """Undo with Ctrl-Z"""
+    do_undo()
+
+
```

Comment:
I think you need to merge main into this file and fix conflicts, these are already on main. Your branch probably wasn't up to date when you started this feature.